docs/1.0/resources/donations.md

```diff
@@ -10,6 +10,7 @@ Default `orderBy`: `createdDateUTC DESC`
 |---|---|---|
 |`/api/participants/{participantID}/donations`|`Array`|To find `{participantID}`, visit your Participant's Fundraising Page. Look for the `&participantID=` URL parameter.|
 |`/api/teams/{teamID}/donations`|`Array`|To find `{teamID}`, visit your Team's Fundraising Page. Look for the `&teamID=` URL parameter.|
+|`/api/events/{eventID}/donations`|`Array`|To find `{eventID}`, visit your Event Landing Page. Look for the `&eventID=` URL parameter.|
```

Comment:
Event's
```suggestion
|`/api/events/{eventID}/donations`|`Array`|To find `{eventID}`, visit your Event's Landing Page. Look for the `&eventID=` URL parameter.|
```

to go inline with the other ones

Reply:
yeah i didnt know what to say here, is "Event Landing Page" even that common a term to PAPI consumers? Event's Landing Page? Event's Fundraising Page?

Reply:
We call it the event landing page.  Not sure how that fits in with "your" though.  In the case of participant and team, it's often "their" fundraising page -- hence the possessive.  Maybe "visit the event landing page" ?

Reply:
ok i updated it to say "Event's Fundraising Page" since that's what it says on the participants endpoint doc